benchmark/CMakeLists.txt

```diff
@@ -95,6 +103,9 @@ endif()
 if(LINUX)
     set(LIBS_OPTIONAL ${LIBS_OPTIONAL} m)
 endif()
+if(UMF_BUILD_GPU_TESTS)
```

Comment:
I think benchmarks are never run on GPU workflow...? you could add this to gpu.yaml or (if it lasts too long) to nigthly build (on gpu runners)

Reply:
@lukaszstolarczuk - let's merge this PR as it is and I can add the GPU benchmark to the workflow in a separate PR

Reply:
we could do that, but at the moment we don't even know if it works, right...?

Reply:
Sorry, folks, I was busy with IPC cache and other stuff. Let me address the comment @lukaszstolarczuk posted.

Reply:
@lukaszstolarczuk Done. Please review.
Also, I excluded `umf-bench-multithreaded` because it takes to long to complete. So I don't think we should waste GPU runner resources.